### PR TITLE
issue no.  #77

### PR DESCRIPTION
**Pull request by trump16**
just removed the print keywords in the setWallpaper.py and schedular,py to avoid random no.

### CODE REVIEW

File: uiplib/setWallpaper.py [hunk 1]
```diff
@@ -25,7 +25,7 @@ def change_windows_background(filename):
 
     SPI_SETDESKWALLPAPER = 0x14  # which command (20)
     SPIF_UPDATEINIFILE = 0x2  # forces instant update
-    print(ctypes.windll.user32.SystemParametersInfoW(SPI_SETDESKWALLPAPER,
+    (ctypes.windll.user32.SystemParametersInfoW(SPI_SETDESKWALLPAPER,
```
Comment (abhsag24) [inline uiplib/setWallpaper.py]: remove the outer brackets

File: uiplib/scheduler.py [hunk 2]
```diff
@@ -11,7 +11,7 @@ class scheduler():
 
     def __init__(self, offline):
         directory = os.path.join(CURR_DIR, PICS_FOLDER)
-        print(time.time())
+        #print(time.time())
```
Comment (abhsag24) [inline uiplib/scheduler.py]: no need to comment. remove it from existence. Terminate it. Let Dogs chew on its bones.

**Comment by abhsag24** [comment]
@trump16 you need to rebase away the first commit(e22115c) that issue is closed

File: uiplib/scheduler.py (hunk 2 above)
Comment (trump16) [inline uiplib/scheduler.py]: ok

**Pull request merged**